pinot-filesystem/src/main/java/com/linkedin/pinot/filesystem/LocalPinotFS.java

```diff
@@ -107,8 +110,9 @@ public long length(URI fileUri) throws IOException {
   }
 
   @Override
-  public void copyToLocalFile(URI srcUri, URI dstUri) throws IOException {
-    copy(srcUri, dstUri);
+  public void copyToLocalFile(URI srcUri, File tempFile) throws Exception {
```

Comment:
suggest rename variable from tempFile to destFile.

It is the application logic that decides whether the file is temp or permanent.

Also, why has IOException changed to Exception?

Reply:
Renamed.

In my next one,  I will add retries, so I am preparing for that.